packages/codegen-ui-react/lib/react-table-renderer-helper.ts

```diff
@@ -0,0 +1,27 @@
+/*
+  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+import { factory, ObjectLiteralElementLike } from 'typescript';
+
+export const objectToLiteralElements = (object: { [key: string]: string | any }): ObjectLiteralElementLike[] => {
```

Comment:
It'd be nice to have a comment showing what the generated output code would look like for these helper functions.

Reply:
Adding comment

Reply:
I think it'd be easier to understand if I knew the output was for something like this
```
stringFormat: {
                  dateTimeFormat: {
                    dateFormat: \\"locale\\",
                    timeFormat: \\"hours24\\",
                  },
                },
or 
stringFormat: 'hello'
```

Reply:
Updating